rest/src/main/java/discord4j/rest/response/ResponseFunction.java

```diff
@@ -107,9 +108,8 @@ static EmptyResponseTransformer emptyOnErrorStatus(RouteMatcher routeMatcher, In
      */
     static RetryingTransformer retryOnceOnErrorStatus(Integer... codes) {
         return new RetryingTransformer(RouteMatcher.any(),
-                reactor.retry.Retry.onlyIf(ClientException.isRetryContextStatusCode(codes))
-                        .fixedBackoff(Duration.ofSeconds(1))
-                        .retryOnce());
+            RetryBackoffSpec.backoff(1, Duration.ofSeconds(1))
```

Comment:
FYI, Retry.onlyIf retries only once, if you wonder if the 1 maxAttempt is the good value.